saleor/static/dashboard-next/AppRoot.tsx

```diff
@@ -438,6 +443,22 @@ export const AppRoot = decorate(
                           menuItems={menuStructure}
                           onMenuItemClick={handleMenuItemClick}
                         />
+                        <div className={classes.spacer} />
+                        <a
+                          className={classes.menuListItem}
+                          href={"/dashboard/next/configuration"}
```

Comment:
Why do we hardcode those? What if we want to change the mount point?

Reply:
Part of the debug code 👎